netmiko/cisco/cisco_xr.py

```diff
@@ -39,6 +39,10 @@ def commit(
         label: str = "",
```

Comment:
Generic comment: we discussed on creating new files and overriding methods in it, so that future netmiko upgrades are easier. did we drop that plan ?

Reply:
Yes, the multiple inheritance was getting too complex.